ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-release-ocm-2.10-presubmits.yaml

```diff
@@ -1015,7 +1015,6 @@ presubmits:
       job-release: "4.15"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-assisted-service-release-ocm-2.10-e2e-ai-operator-ztp-sno-day2-masters
-    optional: true
```

Comment:
these jobs still need to be optional, no?

Reply:
What is the point of having an optional job that is not running unless you trigger it ?

Reply:
without optional=true, you'll need to trigger the job manually to make the check pass

Reply:
You mean without `optional=true`, if you choose to trigger the job and it failed you will have to override it in order to merge ?